package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "github": "0.2.4",
-    "lambda-cfn": "0.1.4",
+    "lambda-cfn": "mapbox/lambda-cfn#update",
```

Comment:
Is it intentional to use the `update` branch of lambda-cfn? Looking at the most recent commits (https://github.com/mapbox/lambda-cfn/commit/44dd6b6047865fb42cb4679c36e01d8cfae599e6 and https://github.com/mapbox/lambda-cfn/commit/8c7c035894b2dbaf652ba7f5d3299feb9b42ed4e) this looks like a debug branch. Can you switch to the latest lambda-cfn, version 0.1.5? This version also enables Node 4.3.2 support, see [the lambda-cfn changelog](https://github.com/mapbox/lambda-cfn/blob/master/CHANGELOG.md#015-2016-12-9).